kettle/update.py

```diff
@@ -38,31 +41,35 @@ def call(cmd):
 
 
 def main():
-    call('time python3 make_db.py --buckets buckets.yaml --junit --threads 32')
+    call('time python3 make_db.py --buckets buckets.yaml --junit --threads {THREADS}')
 
-    bq_cmd = 'bq load --source_format=NEWLINE_DELIMITED_JSON --max_bad_records=1000'
+    bq_cmd = 'bq load --source_format=NEWLINE_DELIMITED_JSON --max_bad_records={MAX_BAD_RECORDS}'
     mj_cmd = 'pypy3 make_json.py'
 
     mj_ext = ''
     bq_ext = ''
     try:
-        call(mj_cmd + ' --days 1 --assert-oldest 1.9')
+        call(mj_cmd + ' --days 1 --assert-oldest {DAYS_OLD}')
     except OSError:
         # cycle daily/weekly tables
         bq_ext = ' --replace'
         mj_ext = ' --reset-emitted'
 
-    call(mj_cmd + mj_ext + ' --days 1 | pv | gzip > build_day.json.gz')
-    call(bq_cmd + bq_ext + ' k8s-gubernator:build.day build_day.json.gz schema.json')
+    if os.getenv('DEPLOYMENT', 'prod') == "staging":
```

Comment:
I get that this means we don't have to change the prod deployment, but it also means someone could accidentally the prod deployment if they used a word other than "staging" for this env var.

I might at least flip it to default to staging, or go further and raise early on if this is an unrecognized value.

Reply:
Sounds good, I was thinking about it and didn't want to change the default behavior. Because in Prod we just use "set image" that ENV isn't there currently and I didn't want to change the deployment.